Dockerfile

```diff
@@ -22,6 +22,7 @@ COPY modules/ modules/
 COPY controller/ controller/
 COPY pkg/ pkg/
 COPY internal/ internal/
+COPY test/mocks test/mocks
```

Comment:
Why do we need to include the mocks in the final Dockerfile? Non test code shouldn't rely on these 🤔